tests/test_executable.py

```diff
@@ -83,7 +83,11 @@ def test_executable(
         ("python", "py-spy"),
         ("ruby", "rbspy"),
         ("php", "phpspy"),
-        ("dotnet", "dotnet-trace"),
+        pytest.param(
+            "dotnet",
+            "dotnet-trace",
+            marks=pytest.mark.xfail(reason="https://github.com/Granulate/gprofiler/issues/697"),
```

Comment:
`, strict=False`
it fails for me locally because its strict

Reply:
Skip it locally then. We will fix the issue in the CI but we will not debug why it does or doesn't reproduce on everyone's machine lol

Reply:
But if it fails sometimes it should be with `strict=False` no?

Reply:
Nope it's determinstic, it's a bug in the build of dotnet-trace, see the ticket https://github.com/Granulate/gprofiler/issues/697

IDK why it doesn't fail on local builds - perhaps because of cached layers? i.e in CI it rebuilds and something changed in our base dotnet image.

Reply:
I ran `docker system prune` before building the image from this branch

Reply:
It doesn't prune the cache layers of buildx